### DISCUSSION TITLE
UDP send on Android phone

### DISCUSSION THREAD

**Discussion by ghost** [jchristn/SimpleUdp#11] [General]
Hello Joel,
sorry bothering but I need an idea.
I use SimpleUdp to comm from Android phone into my home network. The app is written with VS-xamarin-.net-core. At least the tests with the android simulator succeed, I receive the UDP telegram on my windows host. But with a installed APK on my phone nothing comes through. The google playstore app "udp sender/receiver" on the phone works well, and if I have open my app too, the port seems to be in use by my app because the playstore app cannot send any more.
Strange! Do you have any idea what might be wrong ...
Greetings from Germany
Uwe

**Comment by jchristn** [comment]
Hi @ghost I'm very sorry I never received a notification about this issue!  Were you able to resolve it?
